server/repo/repo-http-models/pom.xml

```diff
@@ -0,0 +1,69 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.eclipse.vorto</groupId>
+		<artifactId>repo</artifactId>
+		<version>0.4.0-SNAPSHOT</version>
+	</parent>
+	<artifactId>repo-http-models</artifactId>
+	<packaging>jar</packaging>
+
+	<dependencies>
+		<!-- Vorto DSL Dependencies -->
+		<dependency>
+			<groupId>org.eclipse.vorto</groupId>
```

Comment:
Will remove these dependencies since they are not required right now. Will add them later if the need be